client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -239,31 +270,42 @@ export const PrescriptionLineEditForm: React.FC<
               alignItems="center"
               display="flex"
               flexDirection="row"
-              justifyContent="flex-start"
-              gap={1}
+              gap={5}
             >
               <Grid>
-                <InputLabel style={{ fontSize: 12 }}>
-                  {t('label.issue')}
-                </InputLabel>
-              </Grid>
-              <Grid>
-                <NumericTextInput
-                  autoFocus
-                  disabled={disabled}
-                  value={issueUnitQuantity}
-                  onChange={handleIssueQuantityChange}
-                  min={0}
-                  decimalLimit={2}
-                />
+                <Box display="flex" alignItems="center" gap={1}>
+                  <InputLabel sx={{ fontSize: 12 }}>
+                    {t('label.prescribed-quantity')}
+                  </InputLabel>
+                  <NumericTextInput
+                    autoFocus
+                    disabled={disabled}
+                    value={prescribedQuantity}
+                    onChange={handlePrescribedQuantityChange}
+                    min={0}
+                    decimalLimit={2}
+                  />
+                </Box>
               </Grid>
               <Grid>
-                <InputLabel style={{ fontSize: 12 }}>
-                  {t('label.unit-plural', {
-                    count: issueUnitQuantity,
-                    unit: item?.unitName,
-                  })}
-                </InputLabel>
+                <Box display="flex" alignItems="center" gap={1}>
```

Comment:
I don't think you need the `<Box>` -- all it ends up doing is rendering an additional `<div>` around a single child. 

This is something I've started noticing quite a bit, as it leads to annoying bugs such as this one: https://github.com/msupply-foundation/open-msupply/issues/6370

If you need to add styling, just put it on the `Grid` component, as at the end of the day it's just another `<div>` in the output:

```
 <Grid display="flex" alignItems="center" gap={1}>
```